cli/cmd/cluster.go

```diff
@@ -768,10 +779,38 @@ func createDashboard(awsClient *aws.Client, dashboardName string) error {
 
 	err = awsClient.CreateDashboard(dashboardName, consts.DashboardTitle)
 	if err != nil {
+		fmt.Print("\n\n")
 		return err
 	}
 
 	fmt.Println(" ✓")
 
 	return nil
 }
+
+// createOrClearAPIGateway creates an API gateway for the cluster (or clears an existing one if it already exists)
+func createOrClearAPIGateway(awsClient *aws.Client, clusterName string) error {
```

Comment:
maybe change function name because it clears AND creates the gateway

Reply:
I renamed it to `createOrReplaceAPIGateway`, does that sounds ok to you? My thinking is it creates it if it doesn't exist, and it replaces it with a new one if it did exist.

Reply:
this is better